app/src/main/res/layout/legacy_activity.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:background="@color/glia_base_light_color">
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="EXAMPLE LEGACY APPLICATION"
```

Comment:
Taking into account that it's an example app in our official public repo, it seems to me that we can consider a more informative text. For example, "Example with onActivityResult() API".

Reply:
Comment in the PR header:

_Currently for testing purposes there is a button in MainFragment that opens a android.app.Activity type of view with what we can test this functionality. Before merger that button and view will be deleted._

Reply:
Ah, sorry, I forgot about it.